Doc/library/pathlib.rst

```diff
@@ -46,7 +46,7 @@ Pure paths are useful in some special cases; for example:
 Basic use
 ---------
 
-Importing the main class::
+Importing the main class:
```

Comment:
Please revert this change. The current version looks correct to me and the tests passed when I revert it to use ::.